src/NHSD.BuyingCatalogue.Documents.API/Controllers/DocumentsController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NHSD.BuyingCatalogue.Documents.API.Repositories;
+
+namespace NHSD.BuyingCatalogue.Documents.API.Controllers
+{
+    [Route("api/v1/[controller]")]
+    [ApiController]
+    [Produces("application/json")]
+    [AllowAnonymous]
+    public sealed class DocumentsController : ControllerBase
+    {
+        private readonly IDocumentRepository _documentRepository;
+        private readonly ILogger _logger;
+
+        public DocumentsController(IDocumentRepository documentRepository,
+            ILogger<DocumentsController> logger)
```

Comment:
Entirely optional suggestion, I prefer to give each parameter its own line when the method header spills over multiple lines, e.g.:

```c#
public DocumentsController(
    IDocumentRepository documentRepository,
    ILogger<DocumentsController> logger)
```
I think that's slightly more readable but go with your preference as our standards don't currently cover this kind of detail.

Reply:
👍